types/src/testing.rs

```diff
@@ -41,3 +45,15 @@ pub fn create_expired_transaction(now: Timestamp, test_rng: &mut TestRng) -> Tra
         test_rng,
     )
 }
+
+#[cfg(test)]
+pub fn parse_public_key(arg: &str) -> PublicKey {
+    let escaped = format!("\"{}\"", arg);
+    serde_json::from_str(&escaped).unwrap()
```

Comment:
I think it'd probably be cleaner to use `Value` here
```suggestion
    serde_json::from_value(serde_json::Value::String("")).unwrap()
```